product_docs/docs/biganimal/release/getting_started/01_preparing_azure/01_understanding_qotas_in_azure.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: "Understanding requirements in Azure"
+
+---
+
+The following sections explain BigAnimal's requirements and recommended resource limits in Azure.
+
+## Azure Service Provider registrations
+
+BigAnimal creates and manages some of the Azure resources using service providers. For example, if you want to store keys you require the Microsoft.KeyVault service provider. This service provider offers a resource type called vaults for creating the key vault. For more information, see [Azure resource providers and types](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-providers-and-types). 
```

Comment:
Azure refers to this in their doc as Azure Resource Providers. Do we want to change the name here as well?

Reply:
I think so. We can find and replace all the non-consistent usage and use this term .